man/systemd-sysext.xml

```diff
@@ -69,8 +69,10 @@
     <filename>/var/</filename> included in a system extension image will <emphasis>not</emphasis> appear in
     the respective hierarchies after activation.</para>
 
-    <para>System extension images are strictly read-only, and the host <filename>/usr/</filename> and
-    <filename>/opt/</filename> hierarchies become read-only too while they are activated.</para>
+    <para>System extension images are strictly read-only by default. On mutable host file systems,
+    <filename>/usr/</filename> and <filename>/opt/</filename> hierarchies become read-only while extensions
+    are merged. Mutability can be enabled via the <option>--mutable=</option> option; see "Mutability" below
```

Comment:
maybe: `… are merged, unless mutability is enabled. Mutability may be enabled via…`